tap/api/api.go

```diff
@@ -132,11 +132,17 @@ type MizuEntry struct {
 	EstimatedSizeBytes      int    `json:"-" gorm:"column:estimatedSizeBytes"`
 }
 
+type FullEntryWithPolicy struct {
```

Comment:
I think this type is not used, am I missing something?
@seltonfiuza  @mertyildiran  ?

Reply:
Yes you are right, this is from a really old implementation but that it was present until now, removed

Reply:
Nice